feature-libs/product-configurator/rulebased/rulebased-configurator.module.ts

```diff
@@ -1,13 +1,17 @@
 import { NgModule } from '@angular/core';
 import { RulebasedConfiguratorComponentsModule } from './components/rulebased-configurator-components.module';
 import { RulebasedConfiguratorCoreModule } from './core/rulebased-configurator-core.module';
+import { CpqConfiguratorRestModule } from './cpq/cpq-configurator-rest.module';
+import { CpqConfiguratorOccModule } from './occ/cpq/cpq-configurator-occ.module';
 import { VariantConfiguratorOccModule } from './occ/variant/variant-configurator-occ.module';
 
 @NgModule({
   imports: [
     VariantConfiguratorOccModule,
+    CpqConfiguratorOccModule,
     RulebasedConfiguratorCoreModule,
     RulebasedConfiguratorComponentsModule,
+    CpqConfiguratorRestModule,
```

Comment:
Do we _really_ want by default to bundle and run cpq-related JS code for any customer interested in the commerce product configurator (who imports `RulebasedConfiguratorModule`)? If not, we should allow for easy opt-in, i.e. by importing a separate CPQ module (and for example putting it into customer's app via schematics). What do you think about it?

Reply:
I will look into that

Reply:
@Platonn One way to achieve that: 
- Introduce a new entry point @spartacus/product-configurator/rulebased/cpq that has CPQ rest adapter, converters, maybe also OCC for cart and order integration
- Via schematics bind a different, cpq specific module to our feature, that contains the standard rulebased module + the CPQ specific stuff
```
        productConfiguratorRulebased: {
          module: () =>
            import('@spartacus/product-configurator/rulebased/cpq').then(
              (m) => m.RulebasedConfiguratorCpqModule
            ),
        },
```
- ( Would we then add another option to add-product-configurator? Could you point to an example?)
 
I tried that, would allow us to save about 100 kB for customers not using CPQ.
Is that what you had in mind?

Reply:
Done